### DISCUSSION TITLE
Option to also install Node.js

### DISCUSSION THREAD

**Discussion by Monochromish** [Flow-Launcher/Flow.Launcher#1491] [Ideas]
JavaScript/TypeScript plugins require node to be installed for usage.
I think it would be great to have an option to also install Node.js during Flow Launcher installation, otherwise a system to catch if Node.js is not installed.

**Comment by jjw24** [comment]
Yes, already opened an issue, I just haven't got around to working on it. Implementation and design will be same as setting up python.

**Reply by jjw24** [reply]
https://github.com/Flow-Launcher/Flow.Launcher/issues/1157
